coops-ui/src/registerServiceWorker.ts

```diff
@@ -19,8 +19,10 @@ if (process.env.NODE_ENV === 'production') {
     updatefound () {
       console.log('New content is downloading.')
     },
-    updated () {
+    updated (registration) {
       console.log('New content is available; please refresh.')
+      let confirmationResult = confirm('New content found! Do you want to reload the app?')
+      if (confirmationResult) { registration.waiting.postMessage({ action: 'skipWaiting' }) }
```

Comment:
I prefer 'if' blocks to be on a separate line unless it's really short (like the return on line 37 below, which is ok!).
In fact, I think you'll get a lint error because your have the { } block on the same line as the if.